### PR TITLE
[RU] Add missing sections, fix Gitter badges

### PR DESCRIPTION
**Pull request by teners**
Add "Welcome" and "Following the tutorial at home" sections

### CODE REVIEW

File: ru/README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
 # Руководство Django Girls
 
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/DjangoGirls/tutorial?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Gitter](https://badges.gitter.im/DjangoGirls/tutorial.svg)](https://gitter.im/DjangoGirls/tutorial)
```
Comment (OPersian) [inline ru/README.md]: @teners Why getting rid of UTM params in GET? They could be used for various external services, e.g. Google Analytics

File: ru/README.md [hunk 2]
```diff
@@ -22,12 +30,18 @@
 
 ![Рисунок 0.1](images/application.png)
 
-> Если ты будешь работать над учебником самостоятельно и без тренера, который смог бы помочь при необходимости, то у нас есть для тебя чат: [!\[Gitter\](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/DjangoGirls/tutorial?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge). Мы попросили наших тренеров и успешно прошедших обучение участников заглядывать туда время от времени и помогать остальным! Не стесняйся задавать там свои вопросы!
+> Если ты будешь работать над учебником самостоятельно и без тренера, который смог бы помочь при необходимости, то у нас есть для тебя чат: [![Gitter](https://badges.gitter.im/DjangoGirls/tutorial.svg)](https://gitter.im/DjangoGirls/tutorial). Мы попросили наших тренеров и успешно прошедших обучение участников заглядывать туда время от времени и помогать остальным! Не стесняйся задавать там свои вопросы!
```
Comment (OPersian) [inline ru/README.md]: Same as above, UTM parameters could be used for various external services, e.g. Google Analytics

File: ru/README.md (hunk 1 above)
Comment (OPersian) [reply ru/README.md]: As far as I can see, GA is used to track tutorial pages indeed

```

  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-57505611-7', 'auto', {name: 'microcdn'});
  ga('microcdn.send', 'pageview');
 // ...
```
@aniav please advice if UTM params should be left as is. Probably I'm missing something, an open issue maybe...

Comment (aniav) [reply ru/README.md]: I think we could do Gitter a favour and use UTM params, so they could track where the clicks come from. Let's leave UTM params as they were :)

Comment (teners) [reply ru/README.md]: Oh shoe, sure thing, I'll get UTM stuff back. To be honest, I just copied this piece of markdown from English version, it has no UTM params there :)

**Pull request merged**